modules/performing-a-comparison-between-kubernetes-manifests.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-network-policies.adoc
+:_content-type: CONCEPT
+[id="performing-a-comparison-between-kubernetes-manifests_{context}"]
+= Performing a comparison between Kubernetes manifests
+
+Performing a comparison between Kubernetes manifests is important to identify configuration differences and changes in the specified directories.
```

Comment:
I think it needs a little info here about how this is being used in this case to see the differences in network policies. Something like, "RHACS allows you to see differences in network policies by examining Kubernetes manifests and viewing the differences in allowed connections using the static network policy connectivity difference command." (rewrite as needed) You do explain this later but just a sentence here might be good since it's what the user sees first.